conf/test.config

```diff
@@ -4,20 +4,32 @@
  * -------------------------------------------------
  * Defines bundled input files and everything required
  * to run a fast and simple test. Use as follows:
- *   nextflow run nf-core/proteomicslfq -profile test
+ *   nextflow run nf-core/proteomicslfq -profile test,<docker/singularity/podman>
```

Comment:
```
podman {
    podman.enabled = true
}
```

@apeltzer might already be sufficient, right?
Podman runs rootless, so I don't think that we need to fix the user/group stuff like with Docker.

Reply:
Well, let's try

Reply:
podman is a 1:1 drop in replacement for docker - in the beginning you could simply alias it like this:

```bash
alias docker=podman
```

And then run with the docker profile and get your stuff working ;-) Someone has added podman support to nextflow, though that requires a minimum version of 20.01.0 - also its still listed as incubating feature (non-production ready), so I'd be a bit cautious 😁  https://www.nextflow.io/docs/edge/podman.html

Reply:
We can add a podman profile to nf-core template of course - no harm. That won't be in this release of proteomicslfq I guess then (although you could add one yourself for now and later switch to the template one ;-))

Reply:
Ah hmm.. I added a profile now, let's just hope. And I think we have 20.01 here as minimum anyway.

Reply:
@apeltzer Yes, I know. Just think that it is more consistent to have a podman profile instead of expecting people to alias it as docker :)

I'll open an issue @tools for this.